packages/common/src/types/semanticLayer.ts

```diff
@@ -222,3 +222,23 @@ export function getFilterFieldNamesRecursively(filter: SemanticLayerFilter): {
         ...orFiltersFieldNames,
     ];
 }
+
+// Helper functions to convert between the column names in the query and the column names in the results
```

Comment:
Why are they different? Just wanna understand

Reply:
I it to be warehouse related 🤔